Moved content items between packs

IkaDemisto opened the pull request:
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold 

## Description
- Deprecated `SetSeverityByScore` and moved to deprecated pack.
- Deprecated `BlockIP ` and moved to deprecated pack.
- Deprecated `DataDomainReputation ` and moved to deprecated pack.
- Deprecated `DefaultIncidentClassifier ` and moved to deprecated pack.
- Moved `StaticAnalyze ` to `D2` pack.

yuvalbenshalom commented:
@IkaDemisto just one note about the PR summery: you did not move emailFieldTriggered to D2 pack. not that I think you should. dod you mean something else?

yuvalbenshalom commented on Packs/CommonScripts/Scripts/script-emailFieldTriggered.yml:
```diff
@@ -14,11 +14,14 @@ script: |-
       var body = 'Hello,\nField ' + args.name + ' was changed from ' + args.old + ' to ' + args.new + ' in incident ' + incidents[0].name + '.\n--DBot';
       var subject = 'Field ' + args.name + ' was triggered';
-      executeCommand('SendEmail', {'to':email, 'subject':subject, 'body':body});
+      executeCommand('send-mail', {'to':email, 'subject':subject, 'body':body});
```
did you test that it works?

IkaDemisto replied:
@yuvalbenshalom seems to work. A lot of scripts are executing `send-mail` command, so don't think it should be a problem.

IkaDemisto commented:
> @IkaDemisto just one note about the PR summery: you did not move emailFieldTriggered to D2 pack. not that I think you should. did you mean something else?

Moved `StaticAnalyze` to D2 pack. Updated the summary.

anara123 commented on Packs/D2/pack_metadata.json:
```diff
@@ -1,20 +1,19 @@
+  "name": "D2",
+  "description": "D2 Agent pack",
+  "support": "xsoar",
+  "currentVersion": "1.0.0",
```
The indentation should be 4 spaces.
And why the version is not `1.0.1`?

IkaDemisto replied:
As long as we are moving items between packs, better to stay with initial version. It is causing issue in later step

yuvalbenshalom commented:
@IkaDemisto please resolve the conflict

IkaDemisto commented:
@yuvalbenshalom done
